bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceDialog.java

```diff
@@ -1047,9 +1055,10 @@ private void addDecorationMargin(Control control) {
 		if (!(layoutData instanceof GridData))
 			return;
 		GridData gd = (GridData) layoutData;
-		FieldDecoration dec = FieldDecorationRegistry.getDefault()
+
+		FieldDecoration dec_one = FieldDecorationRegistry.getDefault()
```

Comment:
Either inline this temporary variable or rename it. Don't use snake_case, use camelCase

Reply:
done